packages/donate-button-v4/src/resetCss.ts

```diff
@@ -17,6 +18,7 @@ const resetcss = `
     border: 0;
     font-size: 100%;
     font: inherit;
+    font-family: 'Basis Grotesque Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Roboto, sans-serif;
```

Comment:
Do we actually load our font for the donate-button?

Reply:
https://github.com/everydotorg/donate-button/blob/7e0c81bebee523493f8b872dbb9cdc676a51b849/packages/donate-button-v4/src/index.tsx#L24
https://github.com/everydotorg/donate-button/blob/7e0c81bebee523493f8b872dbb9cdc676a51b849/packages/donate-button-v4/src/loadFonts.ts#L1-L30